### ISSUE TITLE
wezterm doesn't recognize DecPrivateMode 8452

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#1743]
When running in `wezterm`, we draw one of these pretty much whenever we draw a graphic:

```
 2021-06-09T10:39:13.510Z WARN  wezterm_term::terminalstate    > unhandled DecPrivateMode 8452
 2021-06-09T10:39:18.649Z WARN  wezterm_term::terminalstate    > unhandled DecPrivateMode 8452
```

This is sent to ensure the cursor is placed to the right of the emitted sixel (though I only think we need emit it once?). I'm guessing wezterm operates this way without 8452. Don't emit it if it's going to cause a warning.

or maybe we should get @wez to eat 8452; he's proven a pretty righteous dude in interactions thus far

**Comment by dnkl** [comment]
> (though I only think we need emit it once?

Yup. It's a private mode. Once activated, it stays activated until the corresponding `\E[?8452l` sequence is emitted. Or you issue an `XTRESTORE`.

**Comment by WSLUser** [comment]
All good terminals should eat sequences they don't support. VTE for example does this for anything it doesn't specifically support. So if it's a matter of the Wezterm knowing if there's precedent for other terminals, it's been set for over a decade. Even Windows Terminal eats sequences it doesn't support and that's one of the newest ones on the block.

**Comment by wez** [comment]
wezterm logs (and then ignores) unknown sequences in the hope that someone will report them like this; thanks!
(most people don't ever see the logs).

I've opened an issue to do something about this mode.  In general, wezterm aspires to match xterm's behavior by default, and that's likely what I'll do to resolve this.

Regarding emitting only once: yeah, you should only need to emit this once during program init as the mode will stick until disabled or the terminal is reset.

**Comment by wez** [comment]
> All good terminals should eat sequences they don't support. VTE for example does this for anything it doesn't specifically support. So if it's a matter of the Wezterm knowing if there's precedent for other terminals, it's been set for over a decade. Even Windows Terminal eats sequences it doesn't support and that's one of the newest ones on the block.

🙄
